src/Tutorial/ui/doc/advanced/bladeburners.md

```diff
@@ -0,0 +1,3 @@
+# Bladeburners
+
+PLACEHOLDER
```

Comment:
There's a lot of files like this still. This is a *lot* of docs to bite off in one chunk. IMO it would make more sense to delete each RST as it is converted, and do them in batches, but if you want to do them all in one go I'm not gonna stop you.